### PR TITLE
Sleepers can, once again, eject beakers when noone is inside

### PR DESCRIPTION
**Pull request by FluffyGhoster**
(Re)Added the ability to eject beakers from sleepers when noone is inside them.
The eject beaker button now updates depending on if the beaker is installed or not.

### CODE REVIEW

File: tgui/packages/tgui/interfaces/Sleeper.tsx [hunk 1]
```diff
@@ -42,7 +42,16 @@ export const Sleeper = (props, context) => {
         {data.occupant ? (
           <OccupantStatus />
         ) : (
-          <BlockQuote>No occupant detected.</BlockQuote>
+          <Table>
+            <BlockQuote>No occupant detected.</BlockQuote>
+            <Button
+              content={data.beaker < 0 ? 'No Beaker Installed' : 'Eject Beaker'}
```
Comment (NonQueueingMatt) [inline tgui/packages/tgui/interfaces/Sleeper.tsx:48]:
```suggestion
              content={data.beaker < 0 ? 'No Beaker' : 'Eject Beaker'}
```

**Review by NonQueueingMatt — APPROVED** [review]

**Comment by NonQueueingMatt** [comment]
!review

File: html/changelogs/fluffyghost-ejectbeakersleepers.yml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes:
+  - rscadd: "(Re)Added the ability to eject beakers from sleepers when noone is inside them."
+  - rscadd: "The eject beaker button now updates depending on if the beaker is installed or not."
```
Comment (SleepyGemmy) [inline html/changelogs/fluffyghost-ejectbeakersleepers.yml:42]:
```suggestion
  - bugfix: "Fixed ejecting beakers from sleepers when noone is inside them not working."
  - bugfix: "The eject beaker button now updates depending on if the beaker is installed or not."
```

File: html/changelogs/fluffyghost-ejectbeakersleepers.yml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes:
+  - bugfix: "(Re)Added the ability to eject beakers from sleepers when nobody is inside them."
```
Comment (SleepyGemmy) [inline html/changelogs/fluffyghost-ejectbeakersleepers.yml:41]:
```suggestion
  - bugfix: "Readded the ability to eject beakers from sleepers when nobody is inside them."
```

**Review by Arrow768 — APPROVED** [review]

**Pull request merged**